lib/wibox/container/only_on_screen.lua

```diff
@@ -0,0 +1,145 @@
+---------------------------------------------------------------------------
+--
+-- @author Uli Schlachter
+-- @copyright 2017 Uli Schlachter
+-- @classmod wibox.container.only_on_screen
```

Comment:
> It might have low number of interdependences, but it is still there.

It can be used on it's own with some minimal efforts (either from GTK+cairo or SVG like the doc tests). This is in line with your dependency work from a couple weeks back. Should `wibox` keep this "minimal interdependences" and exists on its own or should it keeps getting more dependent on Awesome and WMs concepts?

I think given it already is almost a different project, better keeping it that way.